messages/auth.md

```diff
@@ -0,0 +1,37 @@
+# defaultOrgNotFound
+
```

Comment:
If pressed, I'd vote for using "error.<key>" and "error.<key>.actions", just to be completely clear what these things are.  But honestly, I don't really care.  It's  pretty obvious that "defaultOrgNotFound" is an error.  So I leave that up to you.